components/portfolio/works.js

```diff
@@ -19,16 +19,15 @@ const Works = ({ works }) => (
                   <ul className="portfolio__item-categories">
                     {
                       work.category.main && work.category.main !== 'Other'
-                        ? work.category.main.map((category) => (<li className="portfolio__item-category" key={category}>{category}</li>))
+                        ? work.category.main.map((category) => (<span className="tag" key={category}>{category}</span>))
```

Comment:
why did you change it?

Reply:
In order to reuse existing `.tag` styles. But I came to create separate class scoped to portfolio page. I'll fix it.